RELEASE_NOTES.md

```diff
@@ -1,5 +1,339 @@
+
+### dnf_package
+
+The `dnf_package` resource which provides `package` under the hood on any system shipping with DNF has been greatly refactored to resolve multiple issues.
+
+- The `:lock` action now works on RHEL 8
+- Fixes to prevent attempting to install the same package during each Chef Infra Client run
+
+### git
+
+The `git` resource now fully supports why-run mode.
+
+### msu_package resource improvements
+
+The `msu_package` resource has been improved to better work with Microsoft's cumulative update packages. Newer releases of these packages will not correctly install over previous versions, and we've extended the default timeout for install MSU packages to 60 minutes. Thanks for reporting the timeout issue [@danielfloyd](https://github.com/danielfloyd)
+
+### package
+
+The package resource on macOS and Arch Linux systems now supports passing multiple packages into a single package resource via an array. This allows you to collapse multiple resources into a single resource for simpler coobook authoring and it's also significantly faster as it requires fewer calls to the packaging systems. Thanks for the Arch Linux support [@ingobecker](https://github.com/ingobecker)
+
+Example:
+
+```ruby
+package %w(git curl packer)
+```
+
+### service
+
+The `service` resource has been updated to support newer releases of `update-rc.d` so that it properly disables sys-v init services on Debian Linux distributions. Thanks [@robuye](https://github.com/robuye)
+
+### systemd_unit
+
+The `systemd_unit` resource now respects the `sensitive` property and does not output the content of unit files to the logs if set.
+
+### windows_firewall_rule
+
+The `windows_firewall_rule` resource has been greatly improved thanks to work by [@pschaumburg](https://github.com/pschaumburg) and [@tecracer-theinen](https://github.com/tecracer-theinen).
+
+- New `icmp_type` property, which allows setting the ICMP type when setting up ICMP protocol rules.
+- New `displayname` property, which allows defining the display name of the firewall rule.
+- New `group` property, which allows you to specify that only matching firewall rules of the indicated group association are copied.
+- The `description` property will now update if changed.
+- Fixed setting rules with multiple profiles.
+
+### windows_package
+
+The `windows_package` resource now considers `3010` to be a valid exit code by default. The `3010` exit code means that a package has successfully installed, but requires a reboot.
+
+## YAML Recipes
+
+TODO: Write some stuff
+
+## Custom Resource Improvements
+
+### Improved property require behavior
+
+https://github.com/chef/chef/pull/9688
+
+This should be linked to from the breaking change above
+
+### Resource Partials
+
+https://github.com/chef/chef/pull/9632
+
+### after_resource
+
+A new `after_resource` state has been added to resources that allows you to better control the state information reporting to Chef Automate when a resource converges.
```

Comment:
Should this be "the state of information reported to Chef Automate when a resource converges."

Reply:
No, but what I wrote here is confusing. Let me update that.